pages/mesosphere/dcos/1.14/security/ent/index.md

```diff
@@ -186,16 +186,22 @@ To learn how to do this, see [Controlling user access to services](/mesosphere/d
 
 ## <a name="secrets"></a>Spaces for secrets
 
-The secret path controls which services can access it. If you do not specify a path when storing a secret, any service can access it.
+The secret path controls which services (a service usually is a Marathon application started on DC/OS' Root Marathon)
+can access it. Be aware that any service can access a secret stored at the top level.
 
 Secret paths work in conjunction with service groups to control access. However, you do not need to have service groups to control access to secrets, you can also use the name of the service. The following table provides a few examples to show how it works.
 
 | Secret              | Service                  | Can service access secret? |
 |---------------------|--------------------------|----------------------------|
-| `group/secret`      | `/marathon-user/service` | No                         |
-| `group/secret`      | `/group/hdfs/service`    | Yes                        |
-| `group/hdfs/secret` | `/group/spark/service`   | No                         |
-| `hdfs/secret`       | `/hdfs`                  | Yes                        |
+| `/prod/database-key` | `/marathon-user/my-app`  | No                         |
+| `/bi/s3-key`         | `/bi/hdfs/loader`        | Yes                        |
+| `/bi/hdfs/ec2-key`   | `/bi/spark/executor`     | No                         |
+| `/hdfs/secret`       | `/hdfs`                  | Yes                        |
```

Comment:
I think this should list `Service` to the left of `Secret`, so that `Can service access secret` follows the same pattern.

Also, I would like to see `Yes` and `No` enhanced with a comment, e.g.:
```
Yes - secret is in a parent of service path
```